lms/djangoapps/course_home_api/course_metadata/views.py

```diff
@@ -133,6 +132,7 @@ def get(self, request, *args, **kwargs):
             'celebrations': celebrations,
             'user_timezone': user_timezone,
             'can_view_certificate': certificates_viewable_for_course(course),
+            'disable_progress_graph': course.disable_progress_graph
```

Comment:
@RafayGhafoor I would suggest to move this to progress api view in `lms/djangoapps/course_home_api/progress/views.py` as it is specific to progress tab.